presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -173,7 +174,7 @@
 
     private Duration iterativeOptimizerTimeout = new Duration(3, MINUTES); // by default let optimizer wait a long time in case it retrieves some data from ConnectorMetadata
     private Duration queryAnalyzerTimeout = new Duration(3, MINUTES);
-    private boolean enableDynamicFiltering;
+    private boolean enableDynamicFiltering = true;
```

Comment:
Given the context from @ClarenceThreepwood, let's make the property rename change (since this feature has been implemented and stable since 2020), but leave disabled by default since it may affect certain optimizations.